db/migrate/20160211113430_create_cloud_databases.rb

```diff
@@ -0,0 +1,22 @@
+class CreateCloudDatabases < ActiveRecord::Migration
+  def change
+    create_table :cloud_databases do |t|
+      t.string :name
+      t.string :type
+      t.string :ems_ref
+      t.string :db_engine
+      t.string :status
+      t.string :status_reason
+      t.bigint :storage_size
+      t.bigint :storage_quota
```

Comment:
Wonder if we should use decimal over bigint, like we do in other storage type tables.